plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/DraggableAnnotationController.java

```diff
@@ -97,15 +97,7 @@ void removeAnnotationManager(AnnotationManager annotationManager) {
         }
     }
 
-    void onAnnotationUpdated(Annotation annotation) {
-        if (annotation == draggedAnnotation) {
-            stopDragging(draggedAnnotation, draggedAnnotationManager);
-        }
-    }
-
-    void onSourceUpdated() {
-        stopDragging(draggedAnnotation, draggedAnnotationManager);
-    }
+    void onAnnotationDeleted(Annotation annotation) {}
```

Comment:
Thanks for adapting your PR. Here I had in mind that the drag would be cancelled, as was previously done in `onAnnotationUpdated` – such that a consistent state is kept.

Reply:
Ohh, maybe that's why there is this one unit test failing right now :blush:. Let me try!

Reply:
I changed it, but it doesn't fix the failing unit test. Would you have an idea why it fails? It's again mockito complaining that what it expects to be called is not called.

Reply:
```
com.mapbox.mapboxsdk.plugins.annotation.DraggableAnnotationControllerTest > annotationDragStopSourceUpdateTest FAILED
    Wanted but not invoked:
    dragListener.onAnnotationDragFinished(
        annotation
    );
    -> at com.mapbox.mapboxsdk.plugins.annotation.DraggableAnnotationControllerTest.annotationDragStopSourceUpdateTest(DraggableAnnotationControllerTest.java:129)

    However, there was exactly 1 interaction with this mock:
    dragListener.onAnnotationDragStarted(
        annotation
    );
    -> at com.mapbox.mapboxsdk.plugins.annotation.DraggableAnnotationController.startDragging(DraggableAnnotationController.java:162)
        at com.mapbox.mapboxsdk.plugins.annotation.DraggableAnnotationControllerTest.annotationDragStopSourceUpdateTest(DraggableAnnotationControllerTest.java:129)
```